.rubocop_todo.yml

```diff
@@ -68,7 +68,7 @@ Naming/AccessorMethodName:
     - 'app/controllers/api/v1/accounts_controller.rb'
     - 'app/controllers/api/v1/callbacks_controller.rb'
     - 'app/controllers/api/v1/conversations_controller.rb'
-    - 'app/controllers/passwords_controller.rb'
+    - 'app/controllers/devise_overrides/omniauth_callbacks_controller.rb'
```

Comment:
Can we skip rubocop inline for that specific method instead of adding here ?

Reply:
Noted

Reply:
Done